be/src/exec/chunks_sorter_heap_sort.cpp

```diff
@@ -161,6 +164,57 @@ Status ChunksSorterHeapSort::get_next(ChunkPtr* chunk, bool* eos) {
     return Status::OK();
 }
 
+struct SortRuntimeFilterBuilder {
+    template <LogicalType ptype>
+    JoinRuntimeFilter* operator()(ObjectPool* pool, const ColumnPtr& column, int rid, bool asc) {
```

Comment:
should we give JoinRunTimeFilter a new name?

Reply:
we can refactor it in next PR